.github/workflows/azure.yaml

```diff
@@ -42,7 +48,22 @@ jobs:
           username: ${{ secrets.AZURE_CLIENT_ID }}
           password: ${{ secrets.AZURE_CLIENT_SECRET }}
 
+      - name: Check if image already exists in the registry
+        run: |
+          image=${{ env.CONTAINER_REGISTRY }}/apps/oneseismic:$REVISION_ID
+
+          if docker manifest inspect $image > /dev/null ; then
```

Comment:
You can reduce the line count a lot by either just unconditionally `do docker build -t` (if the overhead is not too bad). Or just do:
```
if not docker manifest inspect $image; then
    docker build -t . 
```
in a single step